graypy/handler.py

```diff
@@ -183,6 +183,8 @@ def make_message_dict(record, debugging_fields, extra_fields, fqdn, localname,
     logging.INFO: 6,
     logging.DEBUG: 7,
 }
+if hasattr(logging, 'NOTICE'):
```

Comment:
Umm.. Yes, looks like that. We can keep it like that or If we are adding `NOTICE` then maybe we should add `VERBOSE` and `SPAM` too. What do you think?